crates/isograph_schema/src/magic_mutation_fields.rs

```diff
@@ -27,13 +33,23 @@ lazy_static! {
     static ref FROM_VALUE_KEY_NAME: ValueKeyName = "from".intern().into();
     static ref TO_VALUE_KEY_NAME: ValueKeyName = "to".intern().into();
 }
-
+#[derive(Deserialize, Eq, PartialEq, Debug)]
 pub struct MagicMutationFieldInfo {
+    #[serde(deserialize_with = "deserialize_stringliteral")]
     path: StringLiteralValue,
     field_map: Vec<FieldMapItem>,
+    #[serde(deserialize_with = "deserialize_stringliteral")]
     field: StringLiteralValue,
 }
 
+fn deserialize_stringliteral<'de, D>(deserializer: D) -> Result<StringLiteralValue, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let value = String::deserialize(deserializer)?;
```

Comment:
Is there a way to avoid allocating an extra string here? Not a blocker if we have to, but it would be ideal to be able to deserialize a string literal from a constant value by just matching and returning the inner StringLiteral

Reply:
Done with the macro here [`529d87c` (#31)](https://github.com/isographlabs/isograph/pull/31/commits/529d87c178cb724e986026b9e738c21c4db0a79e)